releases/src/main.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs;
+use std::path::PathBuf;
+use std::time::Duration;
+
+use anyhow::Context;
+use crates_io_api::AsyncClient;
+use releases::{json, Config, CrateVersion};
+use semver::Version;
+use toml::Table;
+
+const CONFIG: &str = "./releases.json";
+const USER_AGENT: &str = "releases_bot (releases.bot@tobin.cc)";
+const RATE_LIMIT_MILLIS: u64 = 1000;
+
+/// Path to the upcoming release repository.
+const REPO: &str = "/home/tobin/build/github.com/tcharding/rust-bitcoin/04-23-release-0.32.0/";
```

Comment:
In c95324811713f26d391489cdb84a732ddb30b481:

We can't merge this PR with this hardcoded :)

Reply:
Fixed!